pytest_mergify/ci_insights.py

```diff
@@ -209,7 +209,7 @@ def _load_flaky_detection(self) -> None:
 
     def _fetch_existing_test_names(self) -> typing.List[str]:
         if not self.token or not self.repo_name or not self.branch_name:
-            return []
+            raise ValueError("'token', 'repo_name' and 'branch_name' are required")
```

Comment:
Is this error supposed to be shown to end user ? If yes i think we should provide more guidance to fix the issue :)